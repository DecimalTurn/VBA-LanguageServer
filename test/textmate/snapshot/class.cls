VERSION 1.0 CLASS
BEGIN
  MultiUse = -1  'True
END
Attribute VB_Name = "ClassName"
Attribute VB_Description = "Class description goes here"
Attribute VB_GlobalNameSpace = False
Attribute VB_Creatable = False
Attribute VB_PredeclaredId = False
Attribute VB_Exposed = False

Option Explicit
