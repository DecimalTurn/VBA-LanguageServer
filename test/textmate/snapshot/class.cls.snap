>VERSION 1.0 CLASS
#^^^^^^^ source.vba entity.other.attribute-name.block.vba
#       ^ source.vba entity.other.attribute-name.block.vba
#        ^^^ source.vba entity.other.attribute-name.block.vba constant.numeric.vba
#           ^^^^^^^ source.vba entity.other.attribute-name.block.vba
>BEGIN
#^^^^^^ source.vba entity.other.attribute-name.block.vba
>  MultiUse = -1  'True
#^^^^^^^^^^^ source.vba entity.other.attribute-name.block.vba
#           ^ source.vba entity.other.attribute-name.block.vba keyword.operator.comparison.vba
#            ^ source.vba entity.other.attribute-name.block.vba
#             ^^ source.vba entity.other.attribute-name.block.vba constant.numeric.vba
#               ^^^^^^^ source.vba entity.other.attribute-name.block.vba comment.line.apostrophe.vba
>END
#^^^ source.vba entity.other.attribute-name.block.vba
>Attribute VB_Name = "ClassName"
#^^^^^^^^^ source.vba meta.attribute.vba keyword.attribute.vba
#         ^ source.vba meta.attribute.vba
#          ^^^^^^^ source.vba meta.attribute.vba entity.other.attribute-name.vba
#                 ^ source.vba meta.attribute.vba
#                  ^ source.vba meta.attribute.vba keyword.operator.comparison.vba
#                   ^ source.vba meta.attribute.vba
#                    ^^^^^^^^^^^ source.vba meta.attribute.vba string.quoted.double.vba
>Attribute VB_Description = "Class description goes here"
#^^^^^^^^^ source.vba meta.attribute.vba keyword.attribute.vba
#         ^ source.vba meta.attribute.vba
#          ^^^^^^^^^^^^^^ source.vba meta.attribute.vba entity.other.attribute-name.vba
#                        ^ source.vba meta.attribute.vba
#                         ^ source.vba meta.attribute.vba keyword.operator.comparison.vba
#                          ^ source.vba meta.attribute.vba
#                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.vba meta.attribute.vba string.quoted.double.vba
>Attribute VB_GlobalNameSpace = False
#^^^^^^^^^ source.vba meta.attribute.vba keyword.attribute.vba
#         ^ source.vba meta.attribute.vba
#          ^^^^^^^^^^^^^^^^^^ source.vba meta.attribute.vba entity.other.attribute-name.vba
#                            ^ source.vba meta.attribute.vba
#                             ^ source.vba meta.attribute.vba keyword.operator.comparison.vba
#                              ^ source.vba meta.attribute.vba
#                               ^^^^^ source.vba meta.attribute.vba constant.language.boolean.vba
>Attribute VB_Creatable = False
#^^^^^^^^^ source.vba meta.attribute.vba keyword.attribute.vba
#         ^ source.vba meta.attribute.vba
#          ^^^^^^^^^^^^ source.vba meta.attribute.vba entity.other.attribute-name.vba
#                      ^ source.vba meta.attribute.vba
#                       ^ source.vba meta.attribute.vba keyword.operator.comparison.vba
#                        ^ source.vba meta.attribute.vba
#                         ^^^^^ source.vba meta.attribute.vba constant.language.boolean.vba
>Attribute VB_PredeclaredId = False
#^^^^^^^^^ source.vba meta.attribute.vba keyword.attribute.vba
#         ^ source.vba meta.attribute.vba
#          ^^^^^^^^^^^^^^^^ source.vba meta.attribute.vba entity.other.attribute-name.vba
#                          ^ source.vba meta.attribute.vba
#                           ^ source.vba meta.attribute.vba keyword.operator.comparison.vba
#                            ^ source.vba meta.attribute.vba
#                             ^^^^^ source.vba meta.attribute.vba constant.language.boolean.vba
>Attribute VB_Exposed = False
#^^^^^^^^^ source.vba meta.attribute.vba keyword.attribute.vba
#         ^ source.vba meta.attribute.vba
#          ^^^^^^^^^^ source.vba meta.attribute.vba entity.other.attribute-name.vba
#                    ^ source.vba meta.attribute.vba
#                     ^ source.vba meta.attribute.vba keyword.operator.comparison.vba
#                      ^ source.vba meta.attribute.vba
#                       ^^^^^ source.vba meta.attribute.vba constant.language.boolean.vba
>
>Option Explicit
#^^^^^^^^^^^^^^^ source.vba keyword.control.vba
>